test/generated/testproto/grpc/dummy_pb2_grpc.pyi

```diff
@@ -8,61 +8,61 @@ import testproto.grpc.dummy_pb2
 import typing
 
 class DummyServiceStub:
-    """DummyService"""
+    r"""DummyService"""
```

Comment:
This is an actual docstring, and I think that is helpful and will help with IDEs that show them. 👍